public/app/features/search/page/components/SearchView.tsx

```diff
@@ -75,10 +76,13 @@ export const SearchView = ({ showManage, folderDTO, hidePseudoFolders, keyboardE
 
       return (
         <div className={styles.noResults}>
-          <div>No results found for your query.</div>
+          <div>
+            {' '}
```

Comment:
Is this space required here?

Reply:
I believe that was added as a result of me setting up my formatter, will remove it as I don't think it does anything. Thanks for pointing that out.